src/components/organisms/AuthenticationModal/InitialForm/InitialForm.tsx

```diff
@@ -13,18 +13,9 @@ export const InitialForm: FC<InitialFormProps> = ({
 }) => {
   return (
     <div className="initial-form">
-      <div
-        className="btn btn-primary btn-block btn-centered create-account-button"
-        onClick={displayRegisterForm}
-      >
-        Create your account
-      </div>
       <div className="buttons-separator">or</div>
-      <div
-        className="btn btn-block btn-centered login-button"
-        onClick={displayLoginForm}
-      >
-        Log In
```

Comment:
Instead of deleting this, it probably would have been better to comment it out + add an `@debt` comment.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1664/commits/3a4c26a0b571d241b6017166ceeea7fb272e0654